gradoop-common/src/main/java/org/gradoop/common/model/impl/properties/Property.java

```diff
@@ -32,6 +32,36 @@
  */
 public class Property implements Value, Serializable, Comparable<Property> {
 
+  /**
```

Comment:
Same goes for all changes in this class. Probably a helper class which converts those primitives vertex, edge, graphhead, properties into gdlStrings is necessary.